### ISSUE TITLE
[Extension]: PBRemTools issues

### ISSUE THREAD

**Issue by bbecausereasonss** [vladmandic/automatic#1150]
### Issue Description

For some reason PBRemTools also creates 2 tabs, clicking on the second tab in the toolbar simply takes me into the first. Not sure how to remove it.

### Version Platform Description

Chrome, latest. Win11x64

### URL link of the extension

https://github.com/mattyamonaca/PBRemTools

![image](https://github.com/vladmandic/automatic/assets/20264016/b7591eaf-91df-49db-933b-cfb7e8e2ea17)

**Comment by vladmandic** [comment]
as far as i can tell, PBRemTools does not create tab "Canvas Editor" nor i have any idea where "Canvas Editor" comes from. check what other extensions you have installed.
also, if there are issues with extensions, report with extension author in their github and create issue here only if its specific to using extension with this repo.

**Comment by bbecausereasonss** [comment]
Sorry, I selected the wrong one in the image. If you look at PBRemtools, you wll see to the left there is another duplcate entry.. Not happening in Automatic's

**Comment by Gerkinfeltser** [comment]
Hey, I saw this issue a bit as well. I'm not seeing it now though.

Not saying this is what fixed the issue but I did go into settings & saw there were two entries for it in when you click on this "Hidden UI Tabs" field. When I clicked one, they both got selected so I left it unclicked (didn't actually make it/them hidden). I've since restarted and the extra tab seems to have dissappeared. Again, your mileage may vary.

![image](https://github.com/vladmandic/automatic/assets/1477787/0de4532f-21d1-48db-adbb-011c533f510a)
